.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
Most people use 2 spaces for indentation in the front-end. Another big part use 4 spaces to do it. Only few people use tabs because tabs are really unpredictable: 2 or 4 spaces in your editor, 8 spaces in Chrome and GitHub. Also, you cannot shift it easily to indent by some word or variable name if it's necessary.

Anyway, if you want to use tabs, super! Keep eyes on having all the code in the repository with the same style. Currently your `*.scss` files do not follow it.